Source/ASCollectionNode.mm

```diff
@@ -499,12 +499,22 @@ - (void)registerSupplementaryNodeOfKind:(NSString *)elementKind
 
 - (void)performBatchAnimated:(BOOL)animated updates:(void (^)())updates completion:(void (^)(BOOL))completion
 {
-  [self.view performBatchAnimated:animated updates:updates completion:completion];
+  ASDisplayNodeAssertMainThread();
+  if (self.nodeLoaded) {
+    [self.view performBatchAnimated:animated updates:updates completion:completion];
+  } else {
+    if (updates) {
+      updates();
+    }
+    if (completion) {
+      completion(YES);
+    }
+  }
 }
 
 - (void)performBatchUpdates:(void (^)())updates completion:(void (^)(BOOL))completion
 {
-  [self.view performBatchUpdates:updates completion:completion];
+  [self performBatchAnimated:YES updates:updates completion:completion];
```

Comment:
Shouldn't we grab the `animated` param from `[UIView areAnimationsEnabled]` like we are currently doing in `ASCollectionView` ([here](https://github.com/facebook/AsyncDisplayKit/blob/master/Source/ASCollectionView.mm#L789)).